blame.go

```diff
@@ -98,8 +96,10 @@ func Blame(c *object.Commit, path string) (*BlameResult, error) {
 
 // Line values represent the contents and author of a line in BlamedResult values.
 type Line struct {
-	Author string // email address of the author of the line.
-	Text   string // original text of the line.
+	// email address of the author of the line.
```

Comment:
Suggested reword: `Author is the email address of the last author that modified the line`